handlers.go

```diff
@@ -237,12 +237,12 @@ func (s *Server) ServeAPI() error {
 	netw.GET("/peers", s.handleNetPeers)
 	netw.GET("/addrs", s.handleNetAddrs)
 
-	miners := public.Group("/miners")
-	miners.GET("", s.handleAdminGetMiners)
-	miners.GET("/failures/:miner", s.handleGetMinerFailures)
-	miners.GET("/deals/:miner", s.handleGetMinerDeals)
-	miners.GET("/stats/:miner", s.handleGetMinerStats)
-	miners.GET("/storage/query/:miner", s.handleQueryAsk)
+	storageProviders := e.Group("/storage-providers")
```

Comment:
can we configure a reroute so clients that call the old /miners endpoints get rerouted to the /storage-providers endpoints with their request intact?

Reply:
@gmelodie is the intent to have backwards compatibility with v2->v0 api endpoints? I thought I saw somewhere that it would be totally disparate.

Reply:
@neelvirdy we will keep old endpoints the way they are at least for a good while. It's okay to delete them in this PR but we'll have to rebase and put them back in after https://github.com/application-research/estuary/pull/596 is merged. 

Short answer: yes break everything

Reply:
ok gotcha, https://github.com/application-research/estuary/pull/596 is what i was missing. it makes more sense to me to merge #596 before starting the rest of this work so you can incrementally contribute these to `dev` and avoid rebasing conflicts, but defer to you